bookkeeper-server/build.gradle

```diff
@@ -64,9 +66,13 @@ dependencies {
     testImplementation depLibs.junit
     testImplementation depLibs.junitFoundation
     testImplementation depLibs.kerbySimpleKdc
+    testImplementation depLibs.metricsCore
     testImplementation depLibs.mockito
     testImplementation depLibs.powermockJunit
     testImplementation depLibs.powermockMockito
+    testImplementation depLibs.slf4j
+    testImplementation depLibs.slf4jLog4j
```

Comment:
If its already implementation dependency it does not have to be testImplementation dependency.